src/erlzk_conn.erl

```diff
@@ -302,6 +302,8 @@ code_change(_OldVsn, State, _Extra) ->
 %% Internal Functions
 %% ===================================================================
 
+shuffle(L) when length(L) == 1 ->
```

Comment:
.Please use pattern-matching rather than a length/1 call for this check, i.e. `shuffle(L = [_]) -> L;`.